cmd/temporalite/main.go

```diff
@@ -289,3 +311,21 @@ func getPragmaMap(input []string) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+func getDynamicConfigValues(input []string) (map[dynamicconfig.Key][]dynamicconfig.ConstrainedValue, error) {
+	ret := make(map[dynamicconfig.Key][]dynamicconfig.ConstrainedValue, len(input))
+	for _, keyValStr := range input {
+		keyVal := strings.SplitN(keyValStr, "=", 2)
+		if len(keyVal) != 2 {
+			return nil, fmt.Errorf("dynamic config value not in KEY=JSON_VAL format")
+		}
+		key := dynamicconfig.Key(keyVal[0])
```

Comment:
Would there be a way to do any additional validation here eg. to check for typos in key names? It seems like keys are all consts in the dynamicconfig package, so I'm guessing there's not a good way to do this using reflection.